styles.css

```diff
@@ -0,0 +1,5 @@
+@import "components/header.css";
+@import "components/card.css";
+@import "components/bookmark.css";
+@import "components/button.css";
+@import "global.css";
```

Comment:
I think its really cool to structure it that way! I structured according to pages and think my version is much messier so I think I will adapt your version :-D  This looks clean!